vscode/src/debugger/activate.ts

```diff
@@ -88,14 +104,15 @@ function registerCommands(context: vscode.ExtensionContext) {
     ),
   );
 
-  function startDebugging(
+  function startQSharpDebugging(
     resource: vscode.Uri | undefined,
     config: { name: string; [key: string]: any },
     options?: vscode.DebugSessionOptions,
   ) {
     clearCommandDiagnostics();
 
-    if (vscode.debug.activeDebugSession?.type === "qsharp") {
+    const ty = vscode.debug.activeDebugSession?.type;
+    if (ty === "qsharp") {
```

Comment:
Why split this out into multiple statements?

Reply:
leftover from changes that are no longer in the code